com.unity.formats.alembic/Runtime/Scripts/Analytics/AlembicExporterAnalytics.cs

```diff
@@ -53,8 +53,11 @@ internal static AlembicExporterAnalyticsEvent CreateEvent(AlembicRecorderSetting
             {
                 return settings.TargetBranch != null ? settings.TargetBranch.GetComponentsInChildren<T>() : Enumerable.Empty<T>();
             }
-
+#if UNITY_2023_1_OR_NEWER
+            return Object.FindObjectsByType<T>(FindObjectsSortMode.InstanceID);
```

Comment:
Why InstanceID instead of None?

Reply:
So it matches the default behavior.

> Obsolete: This function is obsolete, use [Object.FindObjectsByType](https://docs.unity3d.com/2023.2/Documentation/ScriptReference/Object.FindObjectsByType.html) instead. This replacement allows you to specify whether to sort the resulting array. FindObjectsOfType() always sorts by InstanceID, so calling FindObjectsByType(FindObjectsSortMode.InstanceID) produces identical results. If you specify not to sort the array, the function runs significantly faster, however, the order of the results can change between calls.

https://docs.unity3d.com/2023.2/Documentation/ScriptReference/Object.FindObjectsOfType.html